setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup, find_packages
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+requirements = [
+    'girder>=3.0.0a1'
+]
+
+setup(
+    author='Kitware Inc.',
+    author_email='roni.choudhury@kitware.com',
```

Comment:
I've been using `kitware@kitware.com`, but you can leave this as is.

Reply:
Good idea; see 43c91b2f63ea28b00847417052ba826737010317.